src/assets.ts

```diff
@@ -24,6 +26,10 @@ const assets = async (dir: string) => {
           file.path,
           isScript ? "text/javascript" : contentType,
         );
+
+        if (isScript && file.path.includes("/api/")) {
```

Comment:
This is the best opportunity to collection all the API routes. Doing it here avoids having to parse the raw and transpiled files on each request.